tests/bluetooth/bsim_bt/bsim_test_audio/src/common.h

```diff
@@ -49,10 +49,10 @@
 		bs_trace_error_time_line(__VA_ARGS__); \
 	} while (0)
 
-#define PASS(...) \
+#define PASS(__str) \
 	do { \
 		bst_result = Passed; \
-		bs_trace_info_time(1, __VA_ARGS__); \
+		bs_trace_info_time(1, "PASSED: %s", __str); \
```

Comment:
We'll get a few "PASSED: <X> passed" strings now though, which is fine but does seem overkill with "passed" twice ;)

Reply:
I can live with that.  It is better than getting none.